literature.bib

```diff
@@ -3169,6 +3277,18 @@ @article{MJ:TAOSD14
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2014 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{FMB+:ASE14,
+	author = {Jean{-}R{\'{e}}my Falleri and
+	Flor{\'{e}}al Morandat and Xavier Blanc and Matias Martinez and Martin Monperrus},
+	title = {{Fine-Grained and Accurate Source Code Differencing}},
+	booktitle = ASE,
+	pages = {313--324},
+	year = 2014,
+	tt-tags = {tree-differencing}
```

Comment:
check please

Reply:
removed line break in author names and changed tt-tags to pb-tags